charts/datadog/values.yaml

```
@@ -1310,6 +1310,10 @@ clusterAgent:
   #          username: datadog
   #          password: <YOUR_CHOSEN_PASSWORD>
 
+  # clusterAgent.kubernetesApiserverCheck.disableUseComponentStatus -- Set this to true to disable use_component_status for the kubernetes_apiserver check
+  kubernetesApiserverCheck:
+    disableUseComponentStatus: false
```

The comments need to be set on the option, else it is not taken into account in the README.md
```suggestion
  # clusterAgent.kubernetesApiserverCheck -- correspond to options for configuring the kube_apiserver integration.
  kubernetesApiserverCheck:
    # clusterAgent.kubernetesApiserverCheck.disableUseComponentStatus -- Set this to true to disable use_component_status for the kube_apiserver integration.
    disableUseComponentStatus: false
```